novawallet/Modules/Vote/Governance/ViewModel/ReferendumStatusViewModelFactory.swift

```diff
@@ -133,10 +133,11 @@ extension ReferendumStatusViewModelFactory: ReferendumStatusViewModelFactoryProt
                 )
             }
         case let .deciding(model):
-            if model.isPassing(for: currentBlock), let confirmationUntil = model.confirmationUntil {
```

Comment:
Let's do projected passing in else case as confirming doesn't care of reject logic